### ISSUE TITLE
transparently control apache2_mod_ properties via apache2_module

### ISSUE THREAD

**Issue by bottkv488** [sous-chefs/apache2#600]
# Brief Description

As the v6.0.0-way of enabling modules is achieved via apache2_module IMO it should be possible to "feed" custom properties to the apache_2_module resource directly.

## Current way

```ruby
apache2_module 'proxy' do
  conf false # see https://github.com/sous-chefs/apache2/issues/598
end

apache2_mod_proxy 'proxy' do
  require 'all denied'
  proxy_via 'Off'
end
```

## Suggested way

```ruby
apache2_module 'proxy' do
  mod_conf(
    require: 'all denied',
    proxy_requests: 'Off',
    foo: 'bar'
  )
end
```

**Comment by bottkv488** [comment]
@damacus ,
this might help with https://github.com/sous-chefs/apache2/issues/576 ?

**Comment by damacus** [comment]
Resolved as part of #601
